code/game/machinery/dance_machine.dm

```diff
@@ -400,15 +400,16 @@
 
 
 /obj/machinery/disco/proc/dance4(var/mob/living/M)
-	var/speed = rand(1,3)
+	//var/speed = rand(1,3) // CIT CHANGE
 	set waitfor = 0
-	var/time = 30
+	/*var/time = 30 CIT CHANGE -- replaces dance4 with rapid spinning so that disco balls dont make weird shit happen
```

Comment:
>::::{

Reply:
\>:{{{{{
just make it modify transform instead of resting because @Robustin  is a lazy ass

Reply:
im laaaaaaaaaaaaaaaaaaaaaaaaaaaaaazy